src/include/OSL/oslclosure.h

```diff
@@ -120,20 +120,26 @@ struct OSLEXECPUBLIC ClosureColor {
 /// scaled to add parameters after the end of the struct. Alignment is
 /// set to 16 bytes so that 64 bit pointers and 128 bit SSE types in user
 /// structs have the required alignment.
+#ifdef __CUDACC__
+/// Notice in the OptiX implementation we align this to 8 bytes
```

Comment:
Would another solution be to force the GPU to align addresses to 16 bytes?

I realize that these types probably will not need to travel back and forth between CPU and GPU, but it seems preferable to keep the alignment the same, no ?

Reply:
Remind me why the CPU side is 16? Are we really using SSE on these data and does the occasional one that's not 8-byte aligned instead of 16 really impact performance?

Reply:
Presumably 8 bytes is just the lowest common alignment needed for some data structures on device code, I don't know for sure though. Perhaps @timgrant can share some insight if this is the case and if aligning to 16 bytes would be ok?

Reply:
I think the main concern with increasing the alignment is that it might not make the best use of the closure pool, which is likely to be a limited resource. But that might not matter too much in practice. Would it be problematic to define the alignment differently for CPU and GPU?

Reply:
@lgritz The blender folks use SSE types in their closures and submitted the change to bump the alignment to 16.

@timgrant I think the expedient fix is to keep the code as is (different on cpu/gpu). Though I would still tweak the allocation code to use `alignof` so we only need to change it in one place if we ever change our mind.